app/controllers/sessions_controller.rb

```diff
@@ -21,6 +21,6 @@ def remember_login user
     log_in user
     rmb = params[:session][:remember_me]
     rmb == Settings.remember_me ? remember(user) : forget(user)
-    redirect_to user
+    redirect_back_or user
```

Comment:
what is this men ?

Reply:
cái này em đã khai báo trong sessions/helper, nó dùng để chuyển hướng đến URL được yêu cầu nếu nó tồn tại hoặc chuyển hướng tới URL mặc định.